projects/Mallard/src/components/article/types/article/webview.tsx

```diff
@@ -98,6 +98,7 @@ const WebviewWithArticle = ({
             }}
             ref={_ref}
             onShouldStartLoadWithRequest={onShouldStartLoadWithRequest}
+            allowFileAccessFromFileURLs
```

Comment:
Do we need to set this to true? Since the default is false

Reply:
Good shout, I thought the shorthand syntax set it to true. I will make it explicit. Thank you

Reply:
It is now set @aoifemcl15